### ISSUE TITLE
WebPDF export is freezing

### ISSUE THREAD

**Issue by Plinsboorg** [jupyter/nbconvert#1698]
I tried to get webPDF export working on 2 independent Windows 10 PCs, and I've got 2 identical results:
In picture 1 I click on the export button and the result is the infinitely loading page.
![Picture 1](https://user-images.githubusercontent.com/42300339/147970957-89b361d4-1059-442f-a8d7-848efe991d14.png)

In picture 2 I see the active process which is seem to be running infinitely as well
![Picture 2](https://user-images.githubusercontent.com/42300339/147970944-3d903a2e-7093-442d-9164-a99c8aed59a6.png)

In picture 3 there is a demonstration that the console command is leading to the same infinite freeze
![Picture 3](https://user-images.githubusercontent.com/42300339/147970949-0f474bfa-40b7-4b36-ac3a-742ca4f602e4.png)

I tried --allow-chromium-download. And it seems to me that the chromium was installed.
I tried to copy the git repository and run py.test --pyargs nbconvert.
The result is the freeze demonstrated in picture 4
![Picture 4](https://user-images.githubusercontent.com/42300339/147970951-98f35fbd-ffa7-460a-afa0-ab3fd173d873.png)

I've installed a pyppeteer. Export to PDF and HTML are working fine. 

Steps I followed :
On the Windows 10 machine install python, and octave using installer
Install the jupyter lab using pip install command in CMD
Install the octave_kernel using pip install command in CMD
Install the pandoc using pip install command in CMD
Install the pyppeteer using pip install command in CMD
Install the nbconvert[webpdf] using pip install command in CMD
Run the command
jupyter nbconvert --to webpdf Untitled.ipynb --allow-chromium-download
see the infinite cycle
Run export to webPDF from the interface and see the same infinite cycle

It works the same with every notebook I tried. An empty .ipynb file with an empty cell would be minimal notebook that reproduces this issue. 

**Nbconvert version:** 6.3.0

**Comment by martinRenou** [comment]
I was having the same issue locally on Fedora linux. Cleaning up the browsers installed by pyppeteer (under `/home/martin/.local/share/pyppeteer/local-chromium/` on my laptop) and reinstalling the latest one fixed it:
```
export PYPPETEER_CHROMIUM_REVISION=968410
pyppeteer-install
```

**Comment by Plinsboorg** [comment]
Thank you @martinRenou 
I found the issue thanks to your hint. Reinstalling of the chromium didn't help but reinstalling the overall pyppeteer helped. 
I used the command from their github page
`` pip install -U git+https://github.com/pyppeteer/pyppeteer@dev ``

And it the console I got the following
`` 
Found existing installation: pyppeteer 0.2.6
    Uninstalling pyppeteer-0.2.6:
      Successfully uninstalled pyppeteer-0.2.6
Successfully installed pyppeteer-1.0.2 websockets-10.1
``

This means I initially installed the wrong version.
